administrator/components/com_contact/src/View/Contact/HtmlView.php

```diff
@@ -30,66 +25,59 @@
  *
  * @since  1.6
  */
-class HtmlView extends BaseHtmlView
+class HtmlView extends FormView
 {
     /**
-     * The Form object
+     * Set to true, if saving to menu should be supported
      *
-     * @var  \Joomla\CMS\Form\Form
+     * @var boolean
      */
-    protected $form;
+    protected $supportSaveMenu = true;
 
     /**
-     * The active item
+     * Holds the extension for categories, if available
      *
-     * @var  object
+     * @var string
      */
-    protected $item;
+    protected $categorySection = 'com_contact';
 
     /**
-     * The model state
+     * Constructor
      *
-     * @var  \Joomla\Registry\Registry
-     */
-    protected $state;
-
-    /**
-     * Array of fieldsets not to display
-     *
-     * @var    string[]
+     * @param   array  $config  An optional associative array of configuration settings.
      *
-     * @since  5.2.0
+     * @since   __DEPLOY_VERSION__
      */
-    public $ignore_fieldsets = [];
+    public function __construct(array $config)
+    {
+        if (empty($config['option'])) {
+            $config['option'] = 'com_contact';
+        }
+
+        $config['help_link'] = 'Contacts:_Edit';
+
+        $config['toolbar_icon']   = 'address-book contact';
```

Comment:
```suggestion
        $config['toolbar_icon'] = 'address-book contact';
```
Repeat in other instances.

Reply:
And maybe remove the empty line between this and the previous one and align the `=` of the previous one to this one (after applying Quy's suggestion)?

Reply:
I've allowed myself to fix it with the latest commit.